src/modal/modal-ref.ts

```diff
@@ -40,6 +67,55 @@ export class NgbModalRef {
 	private _hidden = new Subject<void>();
 	private _resolve: (result?: any) => void;
 	private _reject: (reason?: any) => void;
+	private _windowAttributes = [
```

Comment:
Let's make this not private to `ModalRef`, but just a global `const WINDOW_ATTRIBUTES = [...]`